Fantasy-Quest/Assets/Scripts/Minigames/AltarGame/Hand.cs

```diff
@@ -0,0 +1,252 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Common;
+using DG.Tweening;
+using DG.Tweening.Core;
+using DG.Tweening.Plugins.Options;
+using Dialogue;
+using Sirenix.OdinInspector;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Minigames.AltarGame
+{
+    [AddComponentMenu("Scripts/Minigames/AltarGame/Minigames.AltarGame.Hand")]
+    internal class Hand : MonoBehaviour
```

Comment:
Maybe it shoul be separetade in MVC?

Reply:
I tried to unload a little this script, but more separating will creating more difficult system. Any way, i think it can be left in this view and when we will add animations(by skeleton) and more gameplay and sound corrections, system can change in way that i cannot predict now.